express-middlewares/src/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const { languages } = require('./languages');
+const { packages } = require('./packages');
+
+const port = 3000;
+const app = express();
+
+app.use((req, res, next) => {
+  const { path } = req;
```

Comment:
I think you can use "originalUrl" here so that you won't need to add this much logic to get the "pathName".